src/test/resources/template/datademo.json

```diff
@@ -3,11 +3,13 @@
         "nodes": [
             {
                 "id": "create_index",
-                "index_name": "demo"
+                "index_name": "demo",
+                "type": "knn"
```

Comment:
I'm actually deleting my "datademo" json file in #47 and the "demo" JSON is in full template style.

Please write your own JSON test class if you need one, but I don't see where you use this one, so best to just undo this file change.

Reply:
I made this change to test locally but then I populated workflowData in test, so yes can remove these changes.